apps/indexer/src/api/controller/proposals-activity.controller.ts

```diff
@@ -73,9 +73,9 @@ export function proposalsActivity(
                       endBlock: z.string(),
                       timestamp: z.string(),
                       status: z.string(),
-                      forVotes: z.number(), // default 0
-                      againstVotes: z.number(), // default 0
-                      abstainVotes: z.number(), // default 0
+                      forVotes: z.string(),
+                      againstVotes: z.string(),
+                      abstainVotes: z.string(),
```

Comment:
why is it a string right now?

Reply:
I don't see the reason either

Reply:
Serialization problems, too big numbers become 1.2345e+22, and we get an unexpected response type error

Reply:
All these numbers comes as the voting power with 18 decimals, that's why they're big numbers btw